changelogs/current.yaml

```diff
@@ -316,8 +316,8 @@ new_features:
     <arch_overview_wasm_foreign_functions>` for details.
 - area: redis
   change: |
-    Optimized the ``INFO`` command to provide limited metrics with a consolidated cluster view and added ``INFO.SHARD``
-    to retrieve shard-specific metrics.
+    moved the implementation of KEYS,SELECT,ROLE to the cluster scope command framework.
```

Comment:
nit: uppercase at the beginning of each sentence, could drop the second sentence so that changelog looks like this:
```
Moved the implementation of KEYS,SELECT,ROLE to the cluster scope command framework.
```